pkg/drawio/nodesRelations.go

```diff
@@ -0,0 +1,193 @@
+package drawio
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"slices"
+	"strconv"
+
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/common"
+)
+
+//////////////////////////////////////////////////////////////////////
+// setNodesRelations() set for each node N, what are the related nodes to be displayed when the html is filtered by N
+/////////////////////////////////////////////////////////////////////////////
+
+func (data *templateData) setNodesRelations(network TreeNodeInterface) {
+	rel := tnRelations(network)
+	res := map[string]map[string][]string{}
+	for _, node := range data.Nodes {
+		nodeID := strconv.Itoa(int(node.ID()))
+		res[nodeID] = map[string][]string{}
+		nodeRelations := []string{strconv.Itoa(int(data.RootID()))}
+		for _, n := range rel[node] {
+			nodeRelations = append(nodeRelations, strconv.Itoa(int(n.ID())))
+		}
+		res[nodeID]["relations"] = nodeRelations
+		res[nodeID]["graphExplanation"] = []string{"Connectivity graph of " + data.NodeName(node)}
```

Comment:
The string `"Connectivity graph of "` is duplicated for each and every node. Can't we have it in the JavaScript?

Reply:
yes.
however, I assume that in the visibility review to the html files, the graphExplanation will be more complicated.
I want to implement this complication here, and not in the javascript